ros2_control_demo_robot/controllers/rrbot_forward_controller_position_sensor.yaml

```
@@ -0,0 +1,33 @@
+controller_manager:
+  ros__parameters:
+    update_rate: 2  # Hz
+
+    forward_command_controller_position:
+      type: forward_command_controller/ForwardCommandController
+
+    joint_state_controller:
+      type: joint_state_controller/JointStateController
+
+    ur_controllers:
+      type: controller_plugins/ForceTorqueStateController
```

This will not work because there is a package which exports this controller name, i.e.,  `controller_plugins`.

If you want to use example implementation from [here](https://github.com/PickNikRobotics/Universal_Robots_ROS2_Driver/blob/develop/ur_controllers/controller_plugins.xml), you should use "ur_controllers/ForceTorqueStateController" type. 
```suggestion
      type: ur_controllers/ForceTorqueStateController
```

---

I just checked with the suggested changes. It is now failing for ['topic_name' parameter was empty]. I think I understand that adding a new parameter named 'topic_name' will resolve this issue.

Can you suggest what should be the final change here?